spec/helpers/item_seasons_helper_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ItemSeasonsHelper, type: :helper do
+  describe '#rotation' do
+    it 'adds rotation +10 to month with index 12' do
+      expect(helper.rotation(12)).to eq('rotate-plus-10')
+    end
+
+    it 'adds rotation +5 to month with index 1 and 6' do
+      expect(helper.rotation(1 || 6)).to eq('rotate-plus-5')
```

Comment:
**issue:** `1 || 6` will always return `1`. We need one case per integer

Reply:
You could use test tables:
```rb
[1, 6].each do |number|
  it "adds rotation +5 to month with index #{number}" do
    expect(helper.rotation(number)).to eq('rotate-plus-5')
  end
end
```

This is not a very common pattern in ruby (comes from `go`), but it's quite useful. We could even go further and:

```rb
[
  { number: 1, rotation: 'plus-5' },
  { number: 6, rotation: 'plus-5' },
  # all other possibilities
].each do |example|
  it "adds rotation #{example[:rotation]} to month with index #{example[:number]}" do
    expect(helper.rotation(number)).to eq("rotate-#{example[:rotation]}")
  end
end
```

Reply:
> **issue:** `1 || 6` will always return `1`. We need one case per integer

derp! true! Sometimes the simple things still get me 😬